qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -39,17 +39,12 @@
 def _add_pre_post_rotations(
     duration, offsets, rabi_rotations, azimuthal_angles, detuning_rotations
 ):
-    """Adds a pre-post pi.2 rotation at the
-    start and end of the sequence.
+    """Adds a pre and post X rotation at the start and end of the sequence.
 
-    The parameters of the pi/2-pulses are chosen in order to cancel out the
-    product of the pulses in the DSS, so that its total effect in the
-    absence of noise is an identity.
-
-    For a DSS that already produces an identity, this function adds X pi/2-pulses
-    in opposite directions, so that they cancel out. If the DDS produces an X
-    gate, the X pi/2-pulses will be in the same direction. If the DDS produces
-    a Y (Z) gate, the pi/2-pulses are around the Y (Z) axis.
+    Note that with these two pre and post X rotations, the net effect of the DDS does not
+    necessarily have to be an identity. For example, given a CPMG sequence of odd number Y pi
```

Comment:
I think it'd be worth clarifying that the net effect does not have to be an identity, but it *will* always be either an identity or Z pi rotation.